summoner.cabal

```diff
@@ -44,6 +44,12 @@ library
   autogen-modules:     Paths_summoner
 
   ghc-options:         -Wall
+                       -Wincomplete-uni-patterns
+                       -Wincomplete-record-updates
+                       -Wcompat
+                       -Widentities
+                       -Wredundant-constraints
+                       -fhide-source-paths
```

Comment:
Wow, I'm surprised that we added these warnings to `library` and we don't need to fix anything! Our code is already perfect ✨